examples/seq2seq/minify_dataset.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+import fire
+
+def minify(src_dir: Path, dest_dir: Path, n: int):
```

Comment:
Nit: type-hint should be `Union[Path, str]` since you wrap `src_dir` and `dest_dir` in a `Path` after.